beads_email/src/main/java/com/beads/email/config/EmailSenderConfig.java

```diff
@@ -19,7 +19,7 @@
  */
 
 @Configuration
-@PropertySource("email/${env}.properties")
+@PropertySource("email/${env:dev}.properties")
```

Comment:
we need to highlight here that properties file related to email properties.
Something like `email/${env:dev}_email.properties`